src/python/WMCore/WMBS/MySQL/Workflow/GetUnfinishedWorkflows.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+"""
+Module for finding out workflows with open subscriptions
+for production and/or processing tasks.
+"""
+
+from WMCore.Database.DBFormatter import DBFormatter
+
+
+class GetUnfinishedWorkflows(DBFormatter):
+    """
+    If a workflow has any production or processing type subscriptions
+    still unfinished, then it is considered an unfinished/active workflow.
+    """
+
+    sql = """SELECT wmbs_workflow.name, wmbs_workflow.spec
```

Comment:
I might have already seen a GH reporting it, if not, do you feel like opening a dedicated issue for tackling this? I'd rather have a dedicated issue such that, whenever we change how DAOs are defined, that we make it consistency across the code (and potentially involving T0 as well).

Reply:
Yes, it can go to separate issue, and the way to make it backward compatible is the following. The DAO APIs should take optional file parameter and read from a file a template. Then, the refactoring can be done to move SQL statements into templates. I'm fine to have it as a separate issue though.